exec/cnex/cnex.c

```diff
@@ -381,7 +383,7 @@ void exec_raiseLiteral(TExecutor *self, TString *name, TString *info, Number cod
                 cJSON_ArrayForEach(symbols, lines)
                 {
                     jip = cJSON_GetArrayItem(symbols, 0);
-                    if ((uint64_t)jip->valueint == p) {
+                     if (!cJSON_IsInvalid(jip) && (uint64_t)jip->valueint == p) {
```

Comment:
Somehow this got misaligned with an extra leading space.

Reply:
Good catch, there!  Fixed.